From andrea.gelmini@gelma.net  Tue Jun 15 11:58:01 2010
From: Andrea Gelmini <andrea.gelmini@gelma.net>
Date: Sun, 23 May 2010 21:48:24 +0200
Subject: n_gsm.c: removed duplicated #includes
To: andrea.gelmini@gelma.net
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Alan Cox <alan@linux.intel.com>
Message-ID: <1274644105-23285-1-git-send-email-andrea.gelmini@gelma.net>


drivers/char/n_gsm.c: linux/timer.h is included more than once.

Signed-off-by: Andrea Gelmini <andrea.gelmini@gelma.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/char/n_gsm.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/char/n_gsm.c
+++ b/drivers/char/n_gsm.c
@@ -43,7 +43,6 @@
 #include <linux/sched.h>
 #include <linux/interrupt.h>
 #include <linux/tty.h>
-#include <linux/timer.h>
 #include <linux/ctype.h>
 #include <linux/mm.h>
 #include <linux/string.h>
